crates/context/src/tx.rs

```diff
@@ -95,6 +96,44 @@ impl Default for TxEnv {
     }
 }
 
+pub enum DeriveTxTypeError {
```

Comment:
Missing
```
#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Reply:
I have pushed this , with all previous changes as well, do check it out !